src/App.tsx

```diff
@@ -1,61 +1,118 @@
 import './App.scss';
-
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
+import { getTodoId, getUser } from './helper';
 
 export const App = () => {
+  const todos: Todo[] = todosFromServer.map(todo => ({
+    ...todo,
+    user: getUser(todo.userId),
+  }));
+
+  const [currentTodos, setCurrentTodos] = useState(todos);
+  const [newTodoTitle, setNewTodoTitle] = useState('');
+  const [newTodoUserId, setNewTodoUserId] = useState('');
+  const newTodoUser = usersFromServer
+    .find(user => user.id === +newTodoUserId);
+
+  const [error, setError] = useState(false);
+
+  const AddTodo = (event: React.FormEvent<HTMLFormElement>) => {
```

Comment:
good practice to name the event handler as handle[eventName]:smiley:

```suggestion
  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
```